gokart/object_storage.py

```diff
@@ -1,19 +1,47 @@
 from datetime import datetime
 
 import luigi
-import luigi.contrib.gcs
-import luigi.contrib.s3
 from luigi.format import Format
 
-from gokart.gcs_config import GCSConfig
-from gokart.gcs_zip_client import GCSZipClient
-from gokart.s3_config import S3Config
-from gokart.s3_zip_client import S3ZipClient
 from gokart.zip_client import ZipClient
 
+try:
+    from gokart.gcs_config import GCSConfig
+    from gokart.gcs_zip_client import GCSZipClient
+
+    # to avoid warning, import here which means gcs dependencies are exist
+    import luigi.contrib.gcs  # isort: skip
+    GCS_AVAILABLE = True
+except ImportError:
+    GCS_AVAILABLE = False
+
+try:
+    from gokart.s3_config import S3Config
+    from gokart.s3_zip_client import S3ZipClient
+
+    # to avoid warning, import here which means s3 dependencies are exist
+    import luigi.contrib.s3  # isort: skip
+    S3_AVAILABLE = True
+except ImportError:
+    S3_AVAILABLE = False
+
 object_storage_path_prefix = ['s3://', 'gs://']
 
 
+def assert_gcs_available():
+    if GCS_AVAILABLE:
+        return
+
+    raise ImportError('gs:// is not available. Please install gokart[gcs]')
```

Comment:
[nits] I thought adding sample command might be less confusing.

```suggestion
    raise ImportError('gs:// is not available. Please install gokart[gcs] with `pip install gokart[gcs]`')
```